requirements.txt

```diff
@@ -1,3 +1,12 @@
-datajoint>=0.13.0
-element-interface>=0.4.0
-plotly
+datajoint==0.14.0
+element_interface==0.5.0
+ipywidgets==8.0.4
+matplotlib==3.7.0
+mkdocs_gen_files==0.4.0
+nd2==0.5.3
+numpy==1.23.5
+plotly==5.13.1
```

Comment:
Were you able to confirm that this version works for the [Jupyter widget plots](https://github.com/datajoint/element-calcium-imaging/tree/main/element_calcium_imaging/plotting)?  If not, no problem, we can just remove the version pin.